lib/ups.rb

```diff
@@ -0,0 +1,18 @@
+class Ups
+  attr_accessor :ups_audit
+  
+  def initialize
+    @ups_audit = ActiveShipping::UPS.new(login: ENV["UPS_LOGIN"], password: ENV["UPS_PASSWORD"], key: ENV["UPS_KEY"])
+  end
+
+  def find_rates(shipment_params)
+    origin = ActiveShipping::Location.new(country: 'US', state: shipment_params["origin"]["state"], city: shipment_params["origin"]["city"], zip: shipment_params["origin"]["zip"])
+    destination = ActiveShipping::Location.new(country: 'US', state: shipment_params["destination"]["state"], city: shipment_params["destination"]["city"], zip: shipment_params["destination"]["zip"])
+    packages = ActiveShipping::Package.new(shipment_params["packages"]["weight"], [shipment_params["packages"]["height"], shipment_params["packages"]["width"], shipment_params["packages"]["length"]], :units => :imperial, :currency => "USD")
+
```

Comment:
To make these lines a bit more readable, for each of the parameters, you could put them on separate lines. 

`country: `
`state: `
`city: `
`zip: `